executor/utils.py

```diff
@@ -122,5 +122,8 @@ def upload_file(client, report_name, report_id, owner_id):
 
     return client.ns('reporting').reports[report_id].action('upload').post(
         data={'file': {'id': json.loads(media_file)['id']}},
```

Comment:
if you use `payload={'file': {'id': json.loads(media_file)['id']}}` instead of data you don't need to specify the content-type